_posts/2021-06-04-status-update.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "2021-06-04: IAC & DevX Team Status Update"
+author: carabasdaniel
+categories:
+  - team
+  - status
+tags:
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-java_ks#370`][puppetlabs-java_ks-pr-370]: "(MODULES-11067) Fix keytool output parsing", thanks to [durist][durist]
+- [`puppetlabs-kubernetes#519`][puppetlabs-kubernetes-pr-519]: "Allow configuring of waiting times during sa creation", thanks to [ZloeSabo][ZloeSabo]
+- [`puppet-strings#276`][puppet-strings-pr-276]: "README.md: update link to docs", thanks to [kenyon][kenyon]
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-vcsrepo`][puppetlabs-vcsrepo] (`5.0.0`)
```

Comment:
Could you ad a reference somewhere about v4.0.1 being pulled please?

Reply:
```suggestion
- [`puppetlabs-vcsrepo`][puppetlabs-vcsrepo] (`5.0.0`) - this is a re-release of the 4.0.1 code unchanged to signify that this version contains a breaking change. Apologies for the confusion.
```

... or something like this?